content/posts/tclap.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Using TCLAP to parse dumpsys commands"
+date: 2023-09-06T17:49:38+02:00
+draft: false
+author: "The Android Team"
+---
+
+## Introduction
+The `Templatized C++ Command Line Parser` is a c++ header library that is used to parse command line arguments. The developers of this library maintain a manual (https://tclap.sourceforge.net/manual.html) that is easy to use, so be sure to check it out.
+
+## Usage
+In order to use this library to parse your dumpsys commands, you can just copy the following code and adapt the todos to fit your needs
```

Comment:
Any includes needed? Any libraries to add to Android.bp?

Reply:
done